app/controllers/api/v1/cocktails_controller.rb

```diff
@@ -0,0 +1,39 @@
+class Api::V1::CocktailsController < ApplicationController
+  before_action :check_query_param
+
+  def index
+      response = RestClient.get(
```

Comment:
Something to think about - What if you don't get ANY response from the api ??

Reply:
Do we need to add a 400 error message if we don't get any response from the api?